pkg/cmd/oauth-apiserver/cmd_test.go

```diff
@@ -39,3 +54,148 @@ func TestServerStartUp(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestAddFlags(t *testing.T) {
+	// test data
+	fs := pflag.NewFlagSet("addflagstest", pflag.PanicOnError)
+	target := oauthapiserver.NewOAuthAPIServerOptions(nil)
+	target.AddFlags(fs)
+	args := []string{
+		"--shutdown-delay-duration=7s",
+		"--shutdown-send-retry-after=true",
+	}
+
+	// act
+	fs.Parse(args)
+
+	// validate
+	expected := &oauthapiserver.OAuthAPIServerOptions{
+		GenericServerRunOptions: &genericapiserveroptions.ServerRunOptions{
+			MaxRequestsInFlight:         400,
+			MaxMutatingRequestsInFlight: 200,
+			RequestTimeout:              time.Minute,
+			MinRequestTimeout:           1800,
+			ShutdownDelayDuration:       time.Second * 7,
+			JSONPatchMaxCopyBytes:       int64(3 * 1024 * 1024),
+			MaxRequestBodyBytes:         int64(3 * 1024 * 1024),
+			EnablePriorityAndFairness:   true,
```

Comment:
I will address this in a separate PR. I think that we should disable PnF for the aggregated servers.
@tkashem wdyt ?

Reply:
This is preexisting - that is PnF is enabled without this PR.

Reply:
You might save effort if you do this after rebasing to 1.29: https://github.com/kubernetes/kubernetes/pull/118886.

Reply:
sure but how are we going to fix the previous versions ?

Reply:
I will try to backport it to 1.28

Reply:
@benluddy thanks for pointing out the patch, it looks like it actually works!

Reply:
yes, I agree, APF should be disabled